nvidia-dnf.py

```diff
@@ -112,25 +124,30 @@ def sack(self, debug = None):
             except:
                 print('Unable to find matching ' + KERNEL_PKG_REAL + ' package')
 
-            # Get package name
-            kmod_pkg_name = KMOD_PKG_PREFIX + '-' + str(driver.version) + '-' + \
-                    str(kernelpkg.version) + '-' + str(remove_release_dist(kernelpkg.release))
+            # Iterate through drivers in stream
+            for a_driver in available_drivers:
```

Comment:
This loop is equivalent to just setting `kmod_pkg_name` using the last element in `available_drivers`, right?